.github/workflows/run-tests.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Checkout
         uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Parse Node version
-        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        run: echo NVMRC=`cat .nvmrc` >> $GITHUB_OUTPUT
```

Comment:
See comments in https://github.com/alphagov/pay-selfservice/pull/3791/files#diff-7314d0ebbd2e9537ae4889316745b4fd2fa43cb86275c9caae18a86ba228b642R29 - let's avoid using backticks